Xamarin.Forms.Core/VisualElement.cs

```diff
@@ -393,12 +393,6 @@ public double ScaleY
 			set => SetValue(ScaleYProperty, value);
 		}
 
-		public Style Style
-		{
-			get { return (Style)GetValue(StyleProperty); }
-			set { SetValue(StyleProperty, value); }
-		}
-
```

Comment:
?

Reply:
These are hiding properties from the base class. Is that intentional? If so, I'll add them back in and add the missing `new`.

Reply:
I have to check...